recipes/zetcd/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set import_path = "github.com/etcd-io/zetcd" %}
+{% set version = "0.0.5" %}
+
+{% set name = import_path.split('/')[-1] %}
+{% set pkg_src = ('src/'+import_path).replace('etcd-io',
```

Comment:
`pkg_src` controls the directory where we will be cloning the git repo or unpacking the `tar.gz` file. 

I used the fact that we have Jinja2 templates to rename the `etcd-io` directory to `coreos` as you had done in your build scripts.